terraform-build-user/main.tf

```diff
@@ -0,0 +1,18 @@
+module "iam_user" {
+  source = "github.com/cisagov/ami-build-iam-user-tf-module"
+
+  providers = {
+    aws                       = aws
+    aws.images-production-ami = aws.images-production-ami
+    aws.images-staging-ami    = aws.images-staging-ami
+    aws.images-production-ssm = aws.images-production-ssm
+    aws.images-staging-ssm    = aws.images-staging-ssm
+  }
+
+  ssm_parameters = ["/cyhy/dev/users", "/ssh/public_keys/*"]
+  user_name      = "build-skeleton-packer"
```

Comment:
The user name and application need to be updated, and possibly the list of SSM ParameterStore parameters.

Reply:
Fixed in https://github.com/cisagov/nessus-packer/pull/3/commits/aafcf86446a602221f477746b89be07d462dfdc3.  I verified that the list of SSM parameters did not need to be changed.